src/utils/vote-presets.js

```diff
@@ -85,10 +117,91 @@ const getVotePresets = ({ presetParams, setPresetParams, provider }) => {
           .multipliedBy(10 ** decimals)
           .toFixed()
 
-        return {
-          to: settings.contracts.financeVault,
-          calldata: Vault.encodeFunctionData('transfer', params)
+        return [
+          {
+            to: settings.contracts.financeVault,
+            calldata: Vault.encodeFunctionData('transfer', params)
```

Comment:
Can you check if the script changes if you use
```suggestion
            calldata: Vault.interface.encodeFunctionData('transfer', params)
```

Reply:
Not needed because Vault is retreived using ethers.utils.Interface function.